From: Stephen Zander <gibreel@debian.org>
Last-Update: 2002-11-19
Description: Add support to the default paper size
 Uses paper size specified by libpaper instead of defaulting to A4. This
 makes the software integrate better with the system and user
 spectations of it.

diff --git a/html2ps b/html2ps
index 07b79e8..51a60b6 100755
--- a/html2ps
+++ b/html2ps
@@ -28,6 +28,7 @@ eval 'exec perl -S $0 "$@"'
 
 $globrc='/etc/html2psrc';
 $ug='/usr/share/doc/packages/html2ps/html2ps.html';
+$default_paper=`paperconf` || 'a4';
 
 $conf=<<'EOR';
 @html2ps {
@@ -46,7 +47,7 @@ $conf=<<'EOR';
     path: "";
   }
   paper {
-    type: A4;
+    type: $default_paper;
     height: "";
     width: "";
   }
